### PR TITLE
change to expect 204 since 202 causes some strange problems with CORS…

### PR DESCRIPTION
**Pull request by stringfellow**
… (probably)

### CODE REVIEW

**Comment by Velmont** [comment]
Wellwell. The errors are not strange at all :)

You are not sending the CORS headers on the actual response.  You're doing the Preflight correct, but you have to also set Access-Control-Allow-Origin on the actual response.  The checks are stateless.

If you want to have a deep understanding of CORS, the spec is the place to be:  https://fetch.spec.whatwg.org/#concept-http-fetch

 * You get into 3.1 because you need a preflight.

 * 3.1.1 is the actual preflight. You can see that it will in 3.1.2 return Network error if it fails.

 * Everything went OK, you're now on 3.2 ...

 * When you get to 3.5, it does the actual fetch, theeen,

 * In 3.6 you see this:

   > If the CORS flag is set and a CORS check for request and response returns failure, return a network error.


In other words, the PATCH-response itself must also pass a simple CORS check.  Adding:

> Access-Control-Allow-Origin: *

to you response should do it.

**Comment by stringfellow** [comment]
I wish there was an emoji for "hero"
